tests/test_dataframe.py

```diff
@@ -322,10 +322,18 @@ def test_table_distinct(db: gp.Database):
 
 
 def test_table_non_default_schema(db: gp.Database):
-    pg_class = db.create_dataframe(table_name="pg_catalog.pg_class")
+    pg_class = db.create_dataframe(table_name="pg_class", schema="pg_catalog")
```

Comment:
```
gpadmin=# create table "\a""" (id int);
NOTICE:  Table doesn't have 'DISTRIBUTED BY' clause -- Using column named 'id' as the Greenplum Database data distribution key for this table.
HINT:  The 'DISTRIBUTED BY' clause determines the distribution of data. Make sure column(s) chosen are the optimal data distribution key to minimize sk
ew.
CREATE TABLE

df = db.create_dataframe(table_name="\\a\"")
```

Special chars in the table/schema name will still be a problem. But let's solve it in the future.